### ISSUE TITLE
Action ignores expand & other parameters

### ISSUE THREAD

**Issue by wojciech-wieronski** [object/Simple.OData.Client#109]
When using Postman I can craft an HTTP message such as POST http://localhost/Checkins(22)/Default.Checkout()?$expand=Location and both the sideffect works and the singleresult can be expanded.

However, when attempting to do this with the the Simpole.OData.Client, it doesn't work.  Upon registering the BeforeRequest handler, I can see none of the parameters make it to the request.  Here is my sample code:

```
public async Task<Checkin> Checkout (int checkinId)
{
	return await Client.For<Checkin>()
				     .Action ("Checkout")
				     .Key (checkinId)
				     .Expand(x=>x.Location)
				     .ExecuteAsSingleAsync ();
}
```

PS I love this Library.  Works great with Xamarin!

**Comment by object** [comment]
Thank you for the issue report. Can you please mail me the metadata document (you can see it in Postman if you go to $metadata endpoint) and the entity types you defined for Checkin and Location. I can have a look. My email address is vagif.abilov@gmail.com

**Comment by object** [comment]
Please check the version 4.5.2. I believe it should work now.

**Comment by wojciech-wieronski** [comment]
Confirming this was was fixed in 4.5.2.  Thank you!
